server/node.go

```diff
@@ -234,17 +234,15 @@ func (n *Node) initStores(clock *hlc.Clock, engines []engine.Engine) error {
 			}
 			return err
 		}
-		if s.Ident.ClusterID != "" {
-			if s.Ident.StoreID == 0 {
-				return util.Error("cluster id set for node ident but missing store id")
-			}
-			capacity, err := s.Capacity()
-			if err != nil {
-				return err
-			}
-			log.Infof("initialized store %s: %+v", s, capacity)
-			n.lSender.AddStore(s)
+		if s.Ident.ClusterID == "" || s.Ident.NodeID == 0 {
```

Comment:
Sure, added a test. This shouldn't happen unless StoreIdent gets corrupted for some reason.